### DISCUSSION TITLE
Waste projects

### DISCUSSION THREAD

**Discussion by Gribnik2K** [subquery/subql#935] [Ideas]
Over time, it will be quite difficult to choose projects. It seems to me that it is worth adding the ability to delete unnecessary, deleted projects when creating a plan Or leave only projects in the "READY" status

![image](https://user-images.githubusercontent.com/42679698/165647553-cfd4c132-cd32-45c1-9ec0-952d39d4a4bb.png)

**Comment by Gribnik2K** [comment]
![image](https://user-images.githubusercontent.com/42679698/165647969-6d956540-0826-4f00-abf0-5cd7e63d2d36.png)
And here, too, the text is unclear. Like a half ID.

**Comment by heyacherry** [comment]
Hello, @Gribnik2K 
Thanks for the report, will have a look into the issue and UI/UX improvement.

**Comment by lilian79511** [comment]
Ticket is created: SQN-412
